.travis.yml

```diff
@@ -1,7 +1,7 @@
 language: ruby
 
 rvm:
-  - 2.1
+  - 2.3.1
```

Comment:
I'm not that used to Travis, do I just list every version that we want to test for like that?
I tried before without the minor version and that was the [build result](https://travis-ci.org/mirego/activerecord_sane_schema_dumper/builds/145028651). Is there anything else I need to do to make it work™?